src/test_l1_dsHost.c

```diff
@@ -174,8 +174,12 @@ void test_l1_dsHost_negative_dsHostInit(void) {
 
     // Step 02: Call dsHostInit() Attempt to initialize dsHost again
     result = dsHostInit();
-    DS_ASSERT_AUTO_TERM_NUMERICAL(result, dsERR_ALREADY_INITIALIZED);
+ #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.